src/OrchardCore.Modules/OrchardCore.Markdown/GraphQL/MarkdownBodyQueryObjectType.cs

```diff
@@ -33,7 +34,7 @@ private static async Task<object> ToHtml(ResolveFieldContext<MarkdownBodyPart> c
 
             var markdown = ctx.Source.Markdown;
             var templateContext = new TemplateContext();
-            markdown = await liquidTemplateManager.RenderAsync(markdown, templateContext);
+            markdown = await liquidTemplateManager.RenderAsync(markdown, NullEncoder.Default, templateContext);
 
             return Markdig.Markdown.ToHtml(markdown);
         }
```

Comment:
Idem, knowing that the `HtmlBodyQueryObjectType` uses the `htmlEncoder`.

Reply:
My mistake, now I realize that the liquid encoding is only for injected values, so this should definitely be encoded at this level. I will fix it where I changed it.